fastlane/lib/fastlane/actions/get_certificates.rb

```diff
@@ -13,7 +13,6 @@ def self.run(params)
 
         begin
           Cert.config = params # we alread have the finished config
-          Cert.config[:api_key] ||= Actions.lane_context[SharedValues::APP_STORE_CONNECT_API_KEY]
```

Comment:
Not needed anymore because `Cert` will look for existing `Spaceship::ConnectAPI.client` now and use that